SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -91,6 +96,17 @@ private static PlatformApiEndpoint GetApiEndpoint(string region)
             return null; // Use default
         }
 
+        static readonly string ChinaRefreshToken = File.ReadAllText(Path.Combine(Environment.ExpandEnvironmentVariables("%LOCALAPPDATA%"), ".improbable/oauth2/oauth2_refresh_token_cn-production"));
```

Comment:
Do we want to support Mac?

Reply:
I took this code from Platform SDK so this might be a problem there as well.

Reply:
Added a todo and made a ticket for this!